index.html

```diff
@@ -172,43 +171,65 @@ <h2>Glossary</h2>
 
-<p><dfn id="def_document_character_set" class="lint-ignore export">Document character set</dfn>. For XML and HTML (from version 4.0 onwards) the document character set is defined to be the Universal
-    Character Set (UCS) as defined by both ISO/IEC 10646 and Unicode standards. What this means is that the logical model describing how XML and HTML are processed is described in terms of the set of characters defined by
-    Unicode. In practical terms, this means that browsers usually convert all text to  Unicode  internally.</p>
+<p><dfn class="lint-ignore export">Decomposed</dfn>. Decomposed text is usually the result of applying [=Unicode normalization form D=] (NFD), which splits Unicode characters into component parts, typically a base character plus one or more diacritics. However, a decomposed sequence of code points may also be intentionally (or unintentionally) used by a content author where a [=precomposed=] alternative exists.</p>
+
+<p><dfn class="lint-ignore export">Dependent vowel</dfn>. <code class="kw" translate="no">Vowel_dependent</code> is one of the categories in the <code class="kw" translate="no">Indic_Syllabic_Category</code> property set (<a href="https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=[:Indic_Syllabic_Category=Vowel_Dependent:]">see a list</a>). The Unicode Standard definition <a href="https://www.unicode.org/glossary/#dependent_vowel">says</a>: <q cite="https://www.unicode.org/glossary/#dependent_vowel">A symbol or sign that represents a vowel and that is attached or combined with another symbol, usually one that represents a consonant.</q>  Dependent vowels are usually combining marks, but may also be letters (eg. in Thai, or New Tai Lue, which has no combining characters).</p>
+
+<p><dfn class="lint-ignore export">Document character set</dfn>. For XML and HTML (from version 4.0 onwards) the document character set is defined to be the Universal Character Set (UCS) as defined by both ISO/IEC 10646 and Unicode standards. What this means is that the logical model describing how XML and HTML are processed is described in terms of the set of characters defined by Unicode. In practical terms, this means that browsers usually convert all text to  Unicode  internally.</p>
```

Comment:
This one seems kind of pedantic and more relevant to 15~20 years ago? Perhaps:

> **Document character set.** The character set used for processing a document, regardless of what character encoding was used to store it. For XML and HTML (from version 4.0), this is always Unicode. This means that browsers convert all text to Unicode internally and the logical model describing how XML or HTML are processed is described in terms of the set of characters defined by Unicode.

Thoughts?